scripts/ansible/common.yaml

```
@@ -13,6 +13,8 @@ sa_name: "{{ role_name }}"
 role_binding_name: "{{ role_name }}"
 default_file_mode: "0644"
 version: "{{ lookup('env','RELEASE_VERSION')| default('latest', true) }}"
+remove_catalog_version_prefix: "{{ lookup('env','REMOVE_CATALOG_VERSION_PREFIX')| default(false, true) | bool}}"
```

```suggestion
remove_catalog_version_prefix: "{{ lookup('env','REMOVE_CATALOG_VERSION_PREFIX')| default(false) | bool}}"
```

Shouldn't it look like this?

---

According to docs (https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#providing-default-values)
`If you want to use the default value when variables evaluate to false or an empty string you have to set the second parameter to true:`
So I assume the true can stay

---

I think we're both right in this specific case. When using bools we don't care for empty vars as they evaluate to false too. But in other cases your solution might be preferred!